resources/js/components/journal/partials/JournalContentEntry.vue

```diff
@@ -30,6 +30,11 @@
             <h3 class="mb1">
               {{ entry.title }}
             </h3>
+            <p class="mb1">
+              <span class="pr2 f6 avenir">
+                {{ $t('journal.journal_created_at') }} {{ entry.created_at }}
```

Comment:
See suggestion on `journal.php`: here you will use the `date` parameter
```suggestion
                {{ $t('journal.journal_created_at', { date: entry.created_at }) }}
```